PET_Clinic/PET_BUSINESS_TRANSACTIONS.md

```diff
@@ -0,0 +1,72 @@
+# Setup business transactions
+The goal in this part of the workshop is to setup up logical applications and correlate the gathered data to business contexts.
+
+The prerequisite for this part is that you have finished the [setup part](PET_SETUP.md) of the workshop.
+
+### Creating a Business Context
+
+In inspectIT a business context represents an application and consists of application definitions and business transactions definitions. We're starting by creating an application context which relates the data to an application.
+
+At first, you have to switch to the ![Configuration perspective](../images/compass.png?raw=true) *Configuration perspective*. As in all Eclipse-based application, perspective bar is available on the top-right of your application window.
+
+Open the ![Business Context Manager](../images/briefcase.png?raw=true) **Business Context Manager** and click on the *Create Application* button. In the appearing dialog, you can specify the name of the application and assign a description, for example create the application Pet_Clinic_API which represents the AngularJS frontend (API Gateway):
+
+Property | Value
+--- | ---
+Name | Pet_Clinic_API
+Description | Pet Clinic Microservice application
+
+### Creating Application Definitions
+
+Once a business context has been created, it is opened in the editor. Now, we have to specify the application definition. In order to correlate data to an application **application mappings** are used.
+
+Create a new **application mapping** by clicking on the ![Add](../images/add_obj.gif?raw=true) plus icon in the upper right corner of the view.
+
+We know that each microservice pf the Petclinic application is running on different ports. Therefore, a **HTTP Server Port Matching** should be created which matches all requests on a specific port to an application. The Pet_Clinic_API is running on port 8080. Specify this port and save the configuration. Now, all data matching this application definition is correlated to it. You can check this by navigating to the *Data Explorer View* of the api-gateway agent. 
```

Comment:
each microservice **pf** -> of

_You can check this by navigating to the *Data Explorer View* of the api-gateway agent._ I think this is only visible in the invocation sequences, maybe add this info..